### DISCUSSION TITLE
iTwins Endpoint Fetch Projects

### DISCUSSION THREAD

**Discussion by joeyflora** [iTwin/community#62] [Q&A] [answered]
When making a request to the https://api.bentley.com/itwins/?subClass=Project endpoint the resulting project list is different than the list of my projects I can see at connect.bentley.com. The docs say the endpoint "Retrieves a list of iTwins that the calling user is a member of." is that different than what is displayed at connect.bentley.com?

**Comment by davidhjones** [comment]
The connect.bentley.com website "My Projects" tab should be consistent with that is returned in the iTwin Platform API -- "The iTwins that you are a member of". 

Are you seeing differences?

**Reply by joeyflora** [reply]
Yes, there are quite a few projects I can see in the MyProjects tab that aren't pulled in via the request to the API.

**Reply by aruniverse** [reply]
Hey Joey,

Try removing the `?subClass=Project` query param. Does that help return the iTwins you are "missing"?

**Reply by joeyflora** [reply]
I did try that and they still are missing without the `?subClass=Project` query param

**Reply by davidbryant222** [reply, marked as answer]
The API will only return the top 100 unless you set the $top parameter. Does connect.bentley.com return more than 100 projects? Also, could you provide the project id for a project that shows up in the portal but not the API?

**Reply by joeyflora** [reply]
It does seem like the projects that are missing have all been created within the past year.

**Reply by joeyflora** [reply]
That's it! I wasn't providing a $top param so it was using the default 100. Thanks!
